apps/mail/app/(routes)/settings/shortcuts/page.tsx

```
@@ -1,107 +1,118 @@
+              </div>
+            </div>
           ))}
         </div>
       </SettingsCard>
     </div>
   );
 }
 
-function Shortcut({ children, keys, action }: { children: ReactNode; keys: string[]; action: string }) {
-  const [isRecording, setIsRecording] = useState(false);
+function Shortcut({
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Shortcut'. Consider to delete it or rename it.

'Shortcut' is defined here:


(lint/suspicious/noRedeclare)